gems/pending/OpenStackExtract/MiqOpenStackVm/MiqOpenStackImage.rb

```diff
@@ -2,6 +2,7 @@
 require_relative '../../MiqVm/MiqVm'
 
 class MiqOpenStackImage
+  attr_reader :vmConfigFile
```

Comment:
I intend to address MiqVm going forward. I want to make it the common base class, having proper subclasses based on type. When I do that, I can address the naming issues. Given this attr_reader references an existing instance variable, I didn't want to make too many unrelated changes in this PR.